client/coral-admin/src/routes/Moderation/components/Comment.css

```diff
@@ -61,6 +61,7 @@
   line-height: 1.5;
   font-weight: 300;
   margin-bottom: 8px;
+  word-break: break-all;
```

Comment:
Hi @hjanuschka, thanks for this PR.

`word-break: break-all;` breaks in the middle of words even if it doesn't have to. Would `word-wrap: break-word;` also do?